apis/cloud.redhat.com/v1alpha1/jobinvocation_types.go

```diff
@@ -0,0 +1,67 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// JobInvocationSpec defines the desired state of JobInvocation
+type JobInvocationSpec struct {
```

Comment:
Now that I think about it - sorry @BlakeHolifield - I wonder if we should call this a `ClowdJobInvocation` to align it with the others?